actions/actions.py

```diff
@@ -56,11 +56,62 @@ def validate_generic(
             return {field: None}
 
 
-class HealthCheckProfileForm(BaseFormAction):
+class HealthCheckTermsForm(BaseFormAction):
     """HealthCheck form action"""
 
     SLOTS = [
-        "age",
+        "terms",
+    ]
+
+    def name(self) -> Text:
+        """Unique identifier of the form"""
+
+        return "healthcheck_terms_form"
+
+    @classmethod
+    def required_slots(cls, tracker: Tracker) -> List[Text]:
+        for slot in cls.SLOTS:
+            if not tracker.get_slot(slot):
+                return [slot]
+        return []
+
+    def slot_mappings(self) -> Dict[Text, Union[Dict, List[Dict]]]:
+        return {
+            "terms": [self.from_intent(intent="affirm", value="yes"), self.from_text()]
+        }
+
+    def validate_terms(
+        self,
+        value: Text,
+        dispatcher: CollectingDispatcher,
+        tracker: Tracker,
+        domain: Dict[Text, Any],
+    ) -> Dict[Text, Optional[Text]]:
+        if value == "more":
```

Comment:
I'm guessing we'll get this value from the `from_text` mapping, does that give us any normalisation? eg. converting `MORE` to `more`.

If not, maybe we should add a new intent for this, so that the NLU can handle that for us.

Reply:
That's a good point. I'll look at adding this to the NLU.